proprietary/tokens/src/components/amsterdam/form-label.tokens.json

```diff
@@ -5,12 +5,12 @@
       "font-family": { "value": "{amsterdam.typography.font-family}" },
       "font-weight": { "value": "{amsterdam.typography.font-weight.bold}" },
       "spacious": {
-        "font-size": { "value": "{amsterdam.typography.spacious.text-level.7.font-size}" },
-        "line-height": { "value": "{amsterdam.typography.spacious.text-level.7.line-height}" }
+        "font-size": { "value": "{amsterdam.typography.spacious.text-level.6.font-size}" },
```

Comment:
This looks a little small, shouldn't it be level 4?

Reply:
Ah yes 6 is too small. I would say 4 is too big, a form field is such a small section that a label or heading with a font size equal to the control’s content would be fine. But most (not all) designs suggest 4 as well. Updated.